src/qibolab/instruments/qm/controller.py

```diff
@@ -120,6 +126,12 @@ class QMController(Controller):
     """Dictionary containing the :class:`qibolab.instruments.qm.devices.Octave`
     instruments being used."""
 
+    BOUNDS = Bounds(
+        waveforms=MAX_DURATION,
+        readout=MAX_READOUT,
+        instructions=MAX_READOUT,
```

Comment:
For example
```suggestion
        readout=MAX_READOUT,
        instructions=MAX_INSTRUCTIONS,
```
basically breaks the unrolling for QM.

For QM it works after this fix, but I am not sure if other issues exist with other instruments (ie. how well tested it is with all instruments).

Reply:
I don't understand what this would break QM, it should be ignored if you set a high number if it creates trouble, right ?

Reply:
That was definitely a typo, and the problem is exactly that `MAX_INSTRUCTIONS` was a high number, and `MAX_READOUT` a small one.

Reply:
Indeed, it is only a typo but because it was setting maximum instructions as 30, which is a very small number for instructions, it was basically almost never doing unrolling, it was splitting every sequence to a separate batch. And it was a bit tricky to find because `MAX_INSTRUCTIONS` is defined as a much larger number on top but initially it wasn’t clear to me why this was not properly used.

Anyway, I’ll just commit the fix (since it’s trivial) and I guess we can merge this after since it is tested on QM and Zurich.